docs/get-started/connect/testnet.md

```diff
@@ -170,6 +198,15 @@ teku validator-client \
     --validator-keys=<path to key file>:<path to password file>[,<path to key file>:<path to password file>,...]
 ```
 
+# Holesky
+
+```bash
+teku validator-client \
+    --network=goerli                      \
```

Comment:
holesky

Reply:
As discussed I removed Holesky from the testnet topic. Will document this once the network is live.